### PR TITLE
fix: #399 get rid of absolute positioning

### PR DESCRIPTION
**Pull request by Alex89198900**
1. Rid of absolute positioning at columns

### CODE REVIEW

File: mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .verticalContainer {
   display: flex;
+  height: 100%;
   flex-direction: column;
+  justify-content: space-between;
```
Comment (tritonJS826) [inline mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss:5]: hmm, thats dangerous

we should not change this component

File: mw-webapp/src/component/table/Table.module.scss [hunk 2]
```diff
@@ -5,6 +5,7 @@ $cell-max-width: 600px;
 
 .table {
   width: 100%;
+  height: 100%;
```
Comment (tritonJS826) [inline mw-webapp/src/component/table/Table.module.scss:8]: why do we need it?

File: mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx [hunk 3]
```diff
@@ -745,47 +745,53 @@ export const Columns = (props: ColumnsProps) => {
...
-                        position={PositionTooltip.LEFT}
-                      >
-                        <Confirm
-                          trigger={<TrashIcon className={styles.icon} />}
-                          content={<p>
-                            {`Are you sure you want to delete the comment "${comment.description}"?`}
-                          </p>}
-                          onOk={() => deleteComment(comment.uuid)}
-                          okText="Delete"
+            <ol className={styles.numberedList}>
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx:748]: why do we need ol here?

File: mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.module.scss [hunk 4]
```diff
@@ -97,9 +93,9 @@ $listLeftMargin: 20px;
 }
 
 .summarySection {
-  position: absolute;
-  bottom: $gapMedium;
   width: calc(100% - 2 * $gapMedium);
+  margin-top: $gapMedium;
+  margin-bottom: $gapMedium;
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.module.scss:98]: margin is not a gap

File: mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx (hunk 3 above)
Comment (tritonJS826) [reply mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx:748]: i see for now we have deeper nesting, could we get rid of some container

File: mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss:5]: Done

File: mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.module.scss (hunk 4 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.module.scss]: Done

File: mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx (hunk 3 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx:742]: Done

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss (hunk 1 above)
Comment (tritonJS826) [reply mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss]: You added height 100% to the vertical container. But it looks like in project some vertical containers should have different height. 

We should not change this component. Use custom classes to change is behavior locally

Comment (Alex89198900) [reply mw-webapp/src/component/verticalContainer/VerticalContainer.module.scss]: Done

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**